network/src/peer/mod.rs

```diff
@@ -184,18 +192,37 @@ where
                             match maybe_message {
                                 Some(Ok(message)) =>  {
                                     if let Err(err) = self.handle_inbound_message(message, write_reqs_tx.clone()).await {
-                                        warn!("Error in handling inbound message from peer: {}. Error: {:?}",
-                                            self_peer_id.short_str(), err);
+                                        warn!(
+                                            NetworkSchema::new(&self.network_context)
+                                                .connection_metadata(&self.connection_metadata)
```

Comment:
I see that `.connection_metadata` includes the `network_address` in each log, I wonder if that might be too much overhead? Remember that rendered `network_address` can be as long as

```
/dns4/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.elb.us-east-9.amazonaws.com/tcp/9999/ln-noise-ik/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/ln-handshake/0
```

I'm not too worried about these error logs, since they're probably very infrequent; however, the "Starting Peer actor", "Peer actor shutdown", and "Closed connection with Peer" will happen for every connection. Of course, if this is no prob. for the logging backend, then just ignore this 😆

Reply:
Yeah, I think it's a tradeoff we can make.

Can probably log just the PeerId, I might have gone a little overboard with this.

Reply:
I think it's fine to also log the connection id and connection origin, just adding the addr is probably too much though

Reply:
Done, added a separate function to allow logging network address sometimes.